tests/test_git_remotes.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::common::TestEnvironment;
+
+pub mod common;
+
+#[test]
+fn test_git_remotes() {
```

Comment:
Test a `<no URL>` case?

Reply:
I think that can only happen if the URl is not valid UTF-8 and the libgit2 API doesn't seem to let us set such a URL (it takes a `&str`). I even tried to make this test write directly to `.jj/repo/store/git/config` but libgit2 apparently can't even open the repo if the config file has (or at least the URL?) has invalid UTF-8. So I guess we'll have to do without the test for now.